sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1849,7 +1849,12 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				getDao(Person.class).executeRaw("ALTER TABLE person ADD COLUMN externalToken text;");
 				getDao(Event.class).executeRaw("ALTER TABLE events ADD COLUMN externalToken text;");
 
-					// ATTENTION: break should only be done after last version
+			case 260:
+				currentVersion = 260;
+				getDao(Event.class).executeRaw("ALTER TABLE events ADD COLUMN signalEvolutionDate timestamp");
```

Comment:
Not sure whether it's actually needed, but please add a semicolon to the end of the SQL statement just to be safe